microservices/embeddingservice/src/embed_text.py

```diff
@@ -41,40 +41,42 @@ def download_model(
 
 
 class EmbedText:
-    def __init__(self, config):
+    def __init__(self, config, model_config):
```

Comment:
It feels strange to me to pass two configs here, especially as come from the same file.
I would suggest to just pass the config and get the models in the constructor